label/models/label_print.py

```diff
@@ -25,8 +24,8 @@ class LabelPrint(models.Model):
         'ir.actions.act_window', 'Sidebar action', readonly=True,
         help="Sidebar action to make this template available on records "
         "of the related document model")
-    ref_ir_value = fields.Many2one(
-        'ir.values', 'Sidebar button', readonly=True,
+    ref_ir_default = fields.Many2one(
+        'ir.default', 'Sidebar button', readonly=True,
         help="Sidebar button to open the sidebar action")
     model_list = fields.Char('Model List', size=256)
```

Comment:
I think this is never used, could you try to delete every reference and see if it still works?